opentelemetry-api/src/opentelemetry/trace/propagation/__init__.py

```diff
@@ -13,7 +13,22 @@
 # limitations under the License.
 from typing import Optional
 
-from opentelemetry.trace import INVALID_SPAN_CONTEXT, Span, SpanContext
+from opentelemetry import trace as trace_api
+from opentelemetry.context import get_value, set_current, set_value
```

Comment:
`set_current` is unused. I wondered why `pyflakes` caught this and not `pylint` -- it looks like we ignore unused imports in init modules by default.

Reply:
https://github.com/open-telemetry/opentelemetry-python/pull/447